Author: Roland Rosenfeld <roland@debian.org>
Description: Remove comment "For: <user>@<host> (<Realname>)" from PS files
 (generated in genps.c and genpictex.c), because we won't ship personal
 information in output files (Closes: #316382).
Bug-Debian: http://bugs.debian.org/316382

--- a/fig2dev/dev/genpictex.c
+++ b/fig2dev/dev/genpictex.c
@@ -51,7 +51,7 @@
 static char 		*linethick = "1pt";
 static char		*plotsymbol = "\\makebox(0,0)[l]{\\tencirc\\symbol{'160}}";
 static int		cur_thickness = -1;
-static Boolean		anonymous = False;
+static Boolean		anonymous = True;
 
 static void
 genpictex_option(opt, optarg)
--- a/fig2dev/dev/genps.c
+++ b/fig2dev/dev/genps.c
@@ -79,7 +79,7 @@
 static char	tmpeps[PATH_MAX];	/* temp filename for eps when adding tiff preview */
 static char	tmpprev[PATH_MAX];	/* temp filename for ASCII or tiff preview */
 
-static Boolean	anonymous = False;
+static Boolean	anonymous = True;
 int		pagewidth = -1;
 int		pageheight = -1;
 int		width, height;
